tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -16,6 +17,9 @@ restore: |
     fi
     distro_install_build_snapd
 
+    dpkg --set-selections < selections
+    rm -f selections
```

Comment:
I'm not super sure what the intension of this is but as its written now nothing will happen, i.e. dpkg will not perform any installs/removes. If you want to perform actions based on this you need to do a `apt-get dselect-upgrade -y`. But even then I'm not sure this is actually what you want, most likely nothing will happen still because new packages will not be marked as "deinstall".